docs/services/slam/_index.md

```diff
@@ -24,6 +24,13 @@ You can conduct SLAM with data collected live by a [RPlidar](/modular-resources/
 
 ![SLAM map built with Viam of a triangle shaped building.](/services/slam/slam-map-example.png)
 
+## Required Components
+
+{{< cards >}}
+{{< relatedcard link="/components/board/" >}}
+{{< relatedcard link="/components/camera/" >}}
```

Comment:
This requires a particular lidar module https://docs-test.viam.dev/2160/public/modular-resources/examples/rplidar/ .   Try amending the relatedcard template so that you can overwrite the icon and and name. We do probably want to keep the camera icon. Let me know if you need help